assets/src/components/site-scan-context-provider/index.js

```diff
@@ -116,17 +116,11 @@ export function siteScanReducer( state, action ) {
 			};
 		}
 		case ACTION_SCANNABLE_URLS_RECEIVE: {
-			if ( ! action.scannableUrls || action.scannableUrls?.length === 0 ) {
-				return {
-					...state,
-					status: STATUS_COMPLETED,
-				};
-			}
-
+			const hasScannableUrls = action.scannableUrls?.length > 0;
```

Comment:
If `action.scannableUrls` is not an array, then this becomes `null > 0`, right? That evaluates to `false` as desired, but it feels a bit loose.

Reply:
That's correct. We could make is more verbose like this:
```suggestion
			const hasScannableUrls = Array.isArray( action.scannableUrls ) && action.scannableUrls.length > 0;
```